volatility3/framework/symbols/linux/utilities/modules.py

```diff
@@ -346,52 +268,30 @@ def run_modules_scanners(
         Returns:
             Dictionary mapping each plugin to its corresponding result
         """
-
-        module_gatherers = {
-            Modules.source_kernel_identifier: cls.get_kernel_module_info,
-            Modules.source_lsmod_identifier: cls._get_list_modules,
-            Modules.source_sysfs_identifier: cls._get_sysfs_modules,
-        }
+        # Throws ValueError if invalid gatherers sent in
+        Modules._validate_gatherers(caller_wanted_gatherers)
```

Comment:
Don't be mean, let people define their own.  5:P  The API is for everyone, but then it's on them if it breaks because of what they handed it...

Reply:
This is gone now, only for a real check on iterable and subclass